modin/pandas/test/test_io.py

```diff
@@ -1403,6 +1403,14 @@ def test_read_parquet_directory(
             )
             start_row = end_row
         path = make_parquet_dir(dfs_by_filename, row_group_size)
+
+        # There are specific files that PyArrow will try to ignore by default
+        # in a parquet directory. One example are files that start with '_'. Our
+        # previous implementation tried to read all files in a parquet directory,
+        # but we now make use of PyArrow to ensure the directory is valid.
+        with open(os.path.join(path, "_committed_file"), "w+") as f:
```

Comment:
```suggestion
        with open(os.path.join(path, "non_parquet_file"), "w+") as f:
```

Will this work?

Reply:
No, it needs to be one of these (from pyarrow implementation):
```python
def _should_silently_exclude(self, file_name):
        return (file_name.endswith('.crc') or  # Checksums
                file_name.endswith('_$folder$') or  # HDFS directories in S3
                file_name.startswith('.') or  # Hidden files starting with .
                file_name.startswith('_') or  # Hidden files starting with _
                file_name in EXCLUDED_PARQUET_PATHS)
```

Reply:
Ok, this is fine then.